frontend/scripts/preprocess-layer-availaibility.js

```diff
@@ -0,0 +1,91 @@
+const moment = require('moment');
+const fs = require('fs');
+const path = require('path');
+
+process.env.TZ = 'UTC';
```

Comment:
@laurentS I had to add this after noticing that the dates created by the CI were different than the ones created locally...

Reply:
So we probably need to be careful down the line

Reply:
maybe add a comment in the code?